content/post/125-go-ipfs-0-8-0.md

```diff
@@ -0,0 +1,109 @@
+---
+date: 2021-02-19
+url: /2021-02-19-go-ipfs-0-6-0/
+title: go-ipfs 0.8.0, and Remote Pinning, is here!
+author: Adin Schmahmann
+tags: ipfs, go-ipfs, release notes
+header_image: 125-go-ipfs-0-8-0.png
+snippet: See what's new in go-ipfs 0.8.0 (spoiler - pins are easier)!
+---
+
+## What's new in go-ipfs 0.8.0? 
+
+This release is focused on making it easier to work with pins! We have some snazzy new features around being able to ask remote services to pin data for you, and modifying large pin sets is much faster than ever before.
+
+## 🔦 Highlights
+
+### 🧷 Remote pinning services
+
+There is now support for asking remote services to pin data for you.
+
+This features a redesign of how we're thinking about pinning and includes some commonly requested features such as:
+
+ * Pins can have names (and coming soon metadata)
+ * Data can be pinned in either the foreground or background
+ * Pins can be searched for by name, CID, or status
+
+Command-line users benefit from `ipfs pin remote` commands, simplifying remote pinning operations. The built-in pinning service API client also executes all necessary remote calls under the hood. Additionally, HTTP API users now have access to new commands under `/api/v0/pin/remote`.
+
+As long as the service supports the vendor-agnostic [IPFS Pinning Service API](https://ipfs.github.io/pinning-services-api-spec/), it can be used directly in go-ipfs.
```

Comment:
As long as a pinning services supports...?

Reply:
```suggestion
As long as a pinning service supports the vendor-agnostic [IPFS Pinning Service API](https://ipfs.github.io/pinning-services-api-spec/), it can be used directly in go-ipfs.
```